pkg/controller/kappnav/kappnav_controller.go

```diff
@@ -382,6 +382,23 @@ func (r *ReconcileKappnav) Reconcile(request reconcile.Request) (reconcile.Resul
 		return r.ManageError(err, kappnavv1.StatusConditionTypeReconciled, instance)
 	}
 
+	//Create or update the cabundle
+	injectCABundleLabel := map[string]string{"config.openshift.io/inject-trusted-cabundle": "true"}
+	cabundleConfig := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instance.GetName()+ "-" + "ocp-ca-bundle",
+			Namespace: instance.GetNamespace(),
+			Labels: injectCABundleLabel,
+		},
+	}
+	err = r.CreateOrUpdate(cabundleConfig, instance, func() error {
```

Comment:
Should be calling kappnavutils.CustomizeConfigMap(cabundleConfig, instance) here so that the other labels that are on all other resources created by the operator are applied.